www/js/goals.js

```diff
@@ -869,9 +857,9 @@ angular.module('emission.main.goals',['emission.services', 'emission.plugin.logg
         handlePendingRefer();
     };
 
-    if (storage.get(HABITICA_REGISTERED_KEY) == true) {
-        getChallenges();
-    }
+    //if (storage.get(HABITICA_REGISTERED_KEY) == true) {
+       // getChallenges();
```

Comment:
so now we get challenges always, not just when the user is registered into habitica?